vuepress/docs/next/tutorials/compose/content-tutorial.md

```diff
@@ -1,108 +1,89 @@
+-   **Save and Approve**: The content will be saved, approved, and ready for use in an application.
 
--   **All**: All created Content is displayed (Draft, To Approve and
-    Approved/Published)
 
--   **Pending Changes**: All created Content that was saved by only clicking on the
-    **Save** or **Save and Continue** button. The content is not
-    approved and therefore is not ready to be published.
+## Finding Content
 
--   **Ready for Approval**: All created Content that is in **Ready** status,
-    and saved by clicking on the **Save** or **Save and Continue** button.
-    The content is ready for review but has not yet been approved and
-    therefore has not been published.
+Using the `Advanced Filters` feature, you can search for Content by name, type, category, group and status. \
+![Content Filters](./img/content-filters.png)
 
--   **Published**: All created Content that has been saved and approved
-    by clicking on the **Save and Approve** button. This content has been
-    published.
+In the resulting table, contents are listed with these parameters: `Name`, `Author`
+`Code`, `Created by`, `Last Edited`, `Type`, `Status`, `Visibility`, `Group`, `Creation` Date, `Last Modified`, and executable `Actions`. Many of the fields can be deleted from view by unchecking them in the `Additions to the Table of Results` drop-down menu.
 
-![image](./img/Content5.png)
+![Content Table List View](./img/content-table-view.png)
 
-In the table list, contents are listed with several parameters: Name,
-Created by, Last Edited, Type, Created Date, Owner Group, Join Group,
-Status, Restrictions, Code, Actions. Fields can be added or deleted from view by selecting or unselecting them
-in the Columns drop-down menu.
+## Content Actions
 
-![image](./img/Content6.png)
+For all items in the `Content` table, the `Actions` drop-down menu provides these executable options:
 
-## Content Actions
+* `Copy/Paste`: replicate the content
+
+* `Draft Version`: edit the unpublished draft version
 
-For all Content in the list, you have access to a set of actions
-through the **Actions** kebab button, such as:
+*  `Published Version`: edit or revert to a previous published version of the content
 
--   **Edit**: update the data fields in the content definition
+*  `Edit`: update the data fields in the content definition
 
--   **Delete**: remove the content
 
--   **Clone**: replicate the content
 
--   **Publish**: make the content visible
+![Content Actions](./img/content-actions.png)
 
--   **Unpublish**: hide the content
 
-![image](./img/Content7.png)
+## Publish, Unpublish, or Delete Content
 
-All Content in the list can be saved to your local file system by clicking the
-**Download** button and selecting either CSV or XLS format.
+To `Publish`, `Unpublish`, or `Delete` content, apply the buttons above the table for checked items. This will take you to a confirmation page where a summary page displays the usage of the item in your application. 
```

Comment:
would remove comma after `Unpublish`

"apply the buttons" sounds awkward -- click on?

change from future to present tense